services/magnetomer.md

```diff
@@ -0,0 +1,24 @@
+# Magnetometer
+
+    identifier: 0x13029088
+    extends: _sensor
+    camel: magneto
+
+A 3-axis magnetometer.
+
+## Registers
+
+    ro forces @ reading {
+        x: i16.16 F
```

Comment:
shouldn't this be T (for Teslas)? I think we should go for nT and use an integer though (like codal) - eg the MAG3110 has range +-1,000,000nT and resolution of 100nT, so we're easily in the int range even for 100x better sensors and much stronger fields